src/components/Repo/Tabs/Tags.jsx

```diff
@@ -43,7 +43,7 @@ const useStyles = makeStyles(() => ({
 
 export default function Tags(props) {
   const classes = useStyles();
-  const { tags } = props;
+  const { tags, repoName, handleDeleteTag } = props;

```

Comment:
rename "handleDeleteTag" property to "onTagDelete" or something similar. Same thing for "TagCard" component